### ISSUE TITLE
Mapbox / Mapnik installation error - Node.js v7.4.0

### ISSUE THREAD

**Issue by woudsma** [faradayio/tilesplash#29]
Using `npm i tilesplash` on OSX 10.11.6 with node v7.4.0 / npm v4.0.5 produces:

```
node-pre-gyp ERR! Tried to download(403): https://mapbox-node-binary.s3.amazonaws.com/mapnik/v3.5.14/Release/node-v51-darwin-x64.tar.gz
node-pre-gyp ERR! Pre-built binaries not found for mapnik@3.5.14 and node@7.4.0 (node-v51 ABI) (falling back to source compile with node-gyp)
/bin/sh: mapnik-config: command not found
gyp: Call to 'mapnik-config --cflags' returned exit status 127 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:305:16)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
gyp ERR! System Darwin 15.6.0
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "--fallback-to-build" "--module=Users/path/to/tile-server/node_modules/mapnik/lib/binding/node-v51-darwin-x64/mapnik.node" "--module_name=mapnik" "--module_path=Users/path/to/tile-server/node_modules/mapnik/lib/binding/node-v51-darwin-x64"
gyp ERR! cwd Users/path/to/tile-server/node_modules/mapnik
gyp ERR! node -v v7.4.0
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=Users/path/to/tile-server/node_modules/mapnik/lib/binding/node-v51-darwin-x64/mapnik.node --module_name=mapnik --module_path=Users/path/to/tile-server/node_modules/mapnik/lib/binding/node-v51-darwin-x64' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (Users/path/to/tile-server/node_modules/mapnik/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at emitTwo (events.js:106:13)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:885:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
node-pre-gyp ERR! System Darwin 15.6.0
node-pre-gyp ERR! command "/usr/local/bin/node" "Users/path/to/tile-server/node_modules/mapnik/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd Users/path/to/tile-server/node_modules/mapnik
node-pre-gyp ERR! node -v v7.4.0
node-pre-gyp ERR! node-pre-gyp -v v0.6.34
node-pre-gyp ERR! not ok
Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=Users/path/to/tile-server/node_modules/mapnik/lib/binding/node-v51-darwin-x64/mapnik.node --module_name=mapnik --module_path=Users/path/to/tile-server/node_modules/mapnik/lib/binding/node-v51-darwin-x64' (1)
tile-server@1.0.0 Users/path/to/tile-server
├── async@2.3.0  extraneous
├── caching@0.1.4  extraneous (git://github.com/faradayio/node-caching.git#8a06c6f7654c9ba5cfac29037fc404e8ca1a5623)
├── clone@1.0.2  extraneous
├── connect-timeout@1.8.0  extraneous
├── express@4.15.2  extraneous
├── pg@6.1.5  extraneous
├── sphericalmercator@1.0.5  extraneous
└── topojson@1.6.27  extraneous

npm WARN tile-server@1.0.0 No description
npm WARN tile-server@1.0.0 No repository field.
npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "i" "tilesplash"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! code ELIFECYCLE

npm ERR! mapnik@3.5.14 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the mapnik@3.5.14 install script 'node-pre-gyp install --fallback-to-build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the mapnik package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-pre-gyp install --fallback-to-build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs mapnik
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls mapnik
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     Users/path/to/tile-server/npm-debug.log
```

**Comment by woudsma** [comment]
Fixed by using node v6.9.5 instead of v7.x

**Comment by wboykinm** [comment]
Thanks for the report. It does look like [we're stuck with v6 for now](https://github.com/mapnik/node-mapnik#depends).
